tailwind.config.js

```diff
@@ -14,71 +14,82 @@ module.exports = {
       current: 'currentColor',
 
       black: '#324350',
-      white: '#FFFFFF',
-      offWhite: '#F8F9FA',
 
       primary: {
-        xl: '#7cc6dd',
-        l: '#0099d7',
-        DEFAULT: '#1876db',
-        d: '#0c68c0',
-        xd: '#0154ac'
+        100: '#7cc6dd',
+        300: '#0099d7',
+        500: '#1876db', // could also be 500
```

Comment:
What do you mean by `could also be 500`? (Or were these leftover comments from before we decided to use `500` instead of `default`?)